cpp/example_code/elasticfilesystem/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+# Copyright 2010-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Copyright dates should be removed; in other words, it should just be "Copyright Amazon.com, Inc." (and so on).

Reply:
Alternatively, you can replace the entire copyright with this shorter version: 
```
Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
SPDX-License-Identifier: Apache-2.0
```

Reply:
Sure, I will update the copyright notice.

But do (or will) you have a template for that? I saw copyright with dates everywhere, so I added that as well.